averagebeard/golang-api
# Initialize server db

## Summary
* initial setup of service
* basic routes for users with `nil` handlers
* db and route initialization 
* config for database using envs

## Test Plan
* setup a Postgres database as outlined by the README
* run `go build -o golang-api.out && ./golang-api.out`
* `Database is ready.` and `Server is ready.` should print to the console.
